common/views/components/OpeningHours/OpeningHours.js

```diff
@@ -4,6 +4,9 @@ import {placesOpeningHours as places} from '../../../model/opening-hours';
 import {Fragment} from 'react';
 import {formatDate} from '../../../utils/format-date';
 import moment from 'moment';
+import OpeningHoursTable from '../../components/OpeningHoursTable/OpeningHoursTable';
+// $FlowFixMe
+import {upcomingExceptionalDates} from '../../../../server/controllers/content';
```

Comment:
Not sure we should be doing this - should we be copying it over from `server` and then referencing `common` from `server`?